changelog.d/1252.feature.md

```diff
@@ -0,0 +1,16 @@
+Add `--requirement` option to `inject` command.
+
+This reads the list of packages to inject from a text file,
+can be used multiple times,
+and can be used in parallel with dependencies specified
+through the command line.
+The option can be abbreviated to `-r`.
+
+Each line is a separate package specifier with the same syntax as the command line.
+Comments are supported with a `#` prefix.
+Hence, the syntax is a strict subset of the pip [requirements file format][pip-requirements] syntax.
+
+If you require full pip functionality, then use the `runpip` command instead;
+however, the installed packages won't be recognised as "injected".
+
+[pip-requirements]: https://pip.pypa.io/en/stable/reference/requirements-file-format/

```

Comment:
This breaks how the changelog is rendered unfortunately: https://pipx--1252.org.readthedocs.build/1252/changelog/
I think I fixed it for the news fragments in the last version by adding bullet points for everything but the first line, leading it to be rendered as a nested unordered list.

Reply:
Is it also too long?

Reply:
Looking at https://github.com/pypa/pipx/pull/1310/files, I think it should be 1 title paragraph and maybe 1 supporting paragraph.

Reply:
Actually, I don't think it automatically handles multipart log entries. So I've removed all of the detail and moved it to the examples.

However, https://github.com/mayeut/pipx/blob/main/changelog.d/1316.feature.md is broken.

Reply:
Yeah I saw. If you feel like it feel free to fix it. Otherwise, I can do it in a separate PR

Reply:
I'll leave it to you. I think the details need moving like I did here.